gunicorn/app/base.py

```diff
@@ -121,12 +121,14 @@ def load_config_from_module_name_or_filename(self, location):
         if location.startswith("python:"):
             module_name = location[len("python:"):]
             cfg = self.get_config_from_module_name(module_name)
-        else:
-            if location.startswith("file:"):
-                filename = location[len("file:"):]
-            else:
-                filename = location
+        elif location.startswith("file:"):
+            filename = location[len("file:"):]
             cfg = self.get_config_from_filename(filename)
+        else:
+            try:
+                cfg = self.get_config_from_module_name(location)
```

Comment:
if we re-add that feature, wouldn’it be better to add the warning based kn the principle of least astonishment? Espacially since this chang re-add a behaviour removed 2 years ago. thoughts?